### DISCUSSION TITLE
What is the largest landlocked country (by area) in the world?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#757] [Galaxy Trivia] [answered]
*answer with flag emoji*

**Comment by peterbaccaro** [comment]
🇰🇿 - There are currently 44 landlocked countries, the largest by area is Kazakhstan, in Central Asia.

**Comment by SANKARAMDAS** [comment, marked as answer]
🇰🇿 -
Characteristics of landlocked countries
The largest by area is Kazakhstan, in Central Asia, while the smallest is the Vatican, in Europe, surrounded by the capital of Italy, Rome. The most populous landlocked country is Ethiopia, in Africa, while the least populous one is the Vatican.

There are currently 44 landlocked countries and four landlocked [de facto states]. 
Africa has the most landlocked countries, at 16, followed by Europe (14), Asia (12), and South America - 2.
